pbs/usersync.go

```diff
@@ -157,7 +157,9 @@ func OptOut(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 
 	err := VerifyRecaptcha(rr)
 	if err != nil {
-		glog.Infof("Optout failed recaptcha: %v", err)
+		if glog.V(2) {
```

Comment:
Irritating as it is... I think `minloglevel` is only supported by the C++ version of `glog`. The Go docs [don't mention it](https://godoc.org/github.com/golang/glog), and the server fails with an error message if I try to use it.

I don't see any way to filter out `Info` messages in Go's version of glog. Only based on `V(#)` levels